test/pizza-guy.spec.js

```diff
@@ -64,4 +79,40 @@ describe('pizza-guy', function() {
       expect(pizzaGuy.deliver([]).start()).toBe(undefined);
     });
   });
+
+  describe('physical files', function() {
+    it('should call onComplete after all the images where downloaded', function(done) {
+      this.timeout(4000);
```

Comment:
This test simply have bad performance. A theory is that it's because it's writing on disk and creating the buffers, so the execution time is between 1500ms and 2000ms. Mocha has 2000ms as default timeout, making this test to fail randomly. Maybe the performance can be improved removing Sync functions, but until that happen, this line of code is required.